Library/Homebrew/brew.sh

```diff
@@ -88,7 +88,10 @@ case "$HOMEBREW_SYSTEM" in
   Linux)  HOMEBREW_LINUX="1" ;;
 esac
 
+if [[ -z $HOMEBREW_CURL ]]
+then
 HOMEBREW_CURL="/usr/bin/curl"
+fi
```

Comment:
Rather than this fix, I'd prefer to set `HOMEBREW_CURL="/usr/bin/curl"` on Mac and `HOMEBREW_CURL=curl` on Linux.